From 00e17ec315e59aaebdf96bb7498c3256c32caf4c Mon Sep 17 00:00:00 2001
From: user <user@email.com>
Date: Fri, 11 Aug 2017 17:28:35 +0300
Subject: [PATCH] Fixing path

---
 themes/3den.zsh-theme | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/themes/3den.zsh-theme b/themes/3den.zsh-theme
index 1d2c7db..7b44542 100644
--- a/themes/3den.zsh-theme
+++ b/themes/3den.zsh-theme
@@ -1,4 +1,4 @@
-PROMPT=$'%{$fg[white]%}$(~/.rvm/bin/rvm-prompt) %{$fg_bold[cyan]%}%~%{$reset_color%}$(git_prompt_info) %{$fg[cyan]%}%D{[%I:%M:%S]}\
+PROMPT=$'%{$fg[white]%}$(rvm-prompt) %{$fg_bold[cyan]%}%~%{$reset_color%}$(git_prompt_info) %{$fg[cyan]%}%D{[%I:%M:%S]}\
 %{$fg_bold[green]%}%n$%{$reset_color%} '
 
 ZSH_THEME_GIT_PROMPT_PREFIX=" %{$fg[white]%}("
-- 
2.7.4

